components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -771,48 +815,13 @@ void RRTMGPRadiation::run_impl (const double dt) {
       // n2 and co are set to constants and are not handled by trcmix;
       // the rest are handled by trcmix
       real2d tmp2d = subview_2d(m_buffer.tmp2d);
-      const auto gas_mol_weights = m_gas_mol_weights;
       for (int igas = 0; igas < m_ngas; igas++) {
         auto name = m_gas_names[igas];
 
         // We read o3 in as a vmr already
         if (name=="o3") continue;
```

Comment:
This skips the line 
```
m_gas_concs.set_vmr(name, tmp2d);
```
down below. Is this intended? The comment `we read o3 in as a vmr already` makes me think the answer is NO, and we were simply trying to skip the trcmix calculation...